benchmark/syclblas/main.cpp

```diff
@@ -8,40 +8,35 @@
 std::unique_ptr<cli_device_selector> cdsp;
 void free_device_selector() { cdsp.reset(); }
 
-// Declare the executorInstancePtr (it is already `extern` declared in utils.hpp
-// but it must be explicitly declared at least once.)
```

Comment:
declared *exactly* once

Reply:
otherwise there'll be linker errors

Reply:
This comment (and this variable) have been deleted, and there is now a local variable called `executor` instead. Anything you wish to modify here?

Reply:
This comment (and this variable) have been deleted, and there is now a local variable called executor instead. Anything you wish to modify here?